README.md

```diff
@@ -34,76 +34,68 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
 kubectl apply -f ./kubemacpool.yaml
 ```
 
-### Opting-in to kubemacpool service
+### kubemacpool service
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on Pods and VirtualMachines.
+The user can specify whether MAC address will be allocated to all VirtualMachines and Pods by default or not.
+This can be done by patching of the following snippets. patched manifest will be on [testing manifests](config/test/kubemacpool.yaml):
 
-#### How to enable/disable kubemacpool for a namespace
+To set kubemacpool to allocate all VirtualMachines and Pods by default (see [opt-out Mode](#How to opt-out kubemacpool MAC assignment for a namespace in Opt-out mode)):
+```bash
+cp config/default/mutatepods_opt_out_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
+cp config/default/mutatevirtualmachines_opt_out_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
+make generate-test
+```
 
-Kubemacpool is disabled by default on a new namespace.
-To enable kubemacpool on a specific namespace:
+To set kubemacpool to not allocate all VirtualMachines and Pods by default (see [opt-in Mode](#How to opt-in kubemacpool MAC assignment for a namespace in Opt-in mode))):
 ```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocate mutatevirtualmachines.kubemacpool.io=allocate
+cp config/default/mutatepods_opt_in_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
+cp config/default/mutatevirtualmachines_opt_in_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
+make generate-test
+```
+
+**note:** The User can of-course set Pods and VirtualMachines to separate opt-modes by sub-setting the above snippets. 
```

Comment:
This will be rendered as one long paragraph. Please add space between notes or turn them into a list

Reply:
Done